src/control/server/storage/bdev/backend_class_test.go

```diff
@@ -237,29 +204,79 @@ func TestBackend_writeNvmeConfig(t *testing.T) {
 				t.Fatal(err)
 			}
 
+			// replace hostname in wantBuf
+			hn, _ := os.Hostname()
```

Comment:
For what it's worth, the fact that we're having to do this in test seems to indicate the need for some refactoring to make hostname an injectable input to production code rather than sprinkling calls to `os.Hostname()` around.

Reply:
Agreed

Reply:
will attempt to address in the follow-on PR https://github.com/daos-stack/daos/pull/4875